Python/avl.py

```diff
@@ -0,0 +1,276 @@
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+class AVLNode:
+    '''
+    A class used to describe a node in an AVL Tree.
+
+    ...
+
+    Attributes
+    ----------
+    value: Any
+        the node hold's this value.
+    left: AVLNode
+        a pointer to the left child of the current node.
+    right: AVLNode
+        a pointer to the right child of the current node.
+    height: int
+        denotes the height of the current node.
+    '''
+    def __init__(self, value):
+        self.value = value
+        self.left = None
+        self.right = None
+        self.height = 1
+
+    def __repr__(self):
+        return f'Node(value={self.value}, height={self.height})'
+
+class AVL:
+    '''A class to denote an AVL Tree.'''
+    def __init__(self, args=None):
+        self.root = None
+        if args:
```

Comment:
Any description upon the value of args? Maybe move it into __init__ as an optional parameter.

Reply:
Adding the description.